.github/workflows/ci.yml

```
@@ -81,21 +81,29 @@ jobs:
 
       - name: Check formatting
         run: pnpm run format:check
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)